### ISSUE TITLE
Return backcast for Keras model

### ISSUE THREAD

**Issue by mnaylor5** [philipperemy/n-beats#47]
Is there a method to return the backcast for each stack in `nbeats-keras`? I am interested in illustrating the interpretability for a model like in the [n-beats paper](https://arxiv.org/abs/1905.10437) (Fig. 2), but this does not seem straightforward in the Keras implementation. 

The forward method of PyTorch model returns both the forecast and backcast, but I am getting consistently much better models using the Keras trainer. I would appreciate any example or instruction you have to access the interpretability features for this implementation.

Thank you!

**Comment by philipperemy** [comment]
Yes definitely possible and it would make the implementations more consistent. I'll try to prioritize it this week! Thanks for reporting

**Comment by mnaylor5** [comment]
Awesome! Thank you, @philipperemy!

**Comment by philipperemy** [comment]
https://github.com/philipperemy/n-beats/pull/48

**Comment by philipperemy** [comment]
@mnaylor5 done! Released in 1.4.0 (with pip). All you need to do is pass `return_backcast=True` to the predict function of the Keras model.

```python
predictions_keras_backcast = model_keras.predict(x_test, return_backcast=True)
```
